src/credentials/attestation_verifier/DeviceAttestationVerifier.h

```diff
@@ -146,6 +146,25 @@ class AttestationTrustStore
      *
      */
     virtual CHIP_ERROR GetProductAttestationAuthorityCert(const ByteSpan & skid, MutableByteSpan & outPaaDerBuffer) const = 0;
+
+    /**
+     * @brief Look-up a CD cert by SKID
+     *
+     * The implementations of this interface must have access to a set of CDs to trust.
+     *
+     * Interface is synchronous, and therefore this should not be used unless to expose a CD
+     * store that is both fully local and quick to access.
+     *
+     * @param[in] skid Buffer containing the subject key identifier (SKID) of the CD to look-up
+     * @param[in,out] outCDDerBuffer Buffer to receive the contents of the CDD cert, if found.
+     *                               Size will be updated to match actual size.
+     *
+     * @returns CHIP_NO_ERROR on success, CHIP_INVALID_ARGUMENT if `skid` or `outCDDerBuffer` arguments
```

Comment:
NOT_IMPLEMENTED seems to be an important return code since most seem to use it. Could we document that too as callers should generally handle it?

Reply:
DeviceAttestationVerifier is a base class never used directly